ERR_MODULE_NOT_FOUND in drizzle adapter

### Adapter type

@auth/drizzle-adapter

### Environment

  System:
    OS: Windows 11 10.0.22631
    CPU: (24) x64 12th Gen Intel(R) Core(TM) i9-12900KF
    Memory: 42.27 GB / 63.83 GB
  Binaries:
    Node: 18.18.2 - ~\scoop\apps\nvm\current\nodejs\nodejs\node.EXE
    npm: 9.8.1 - ~\scoop\apps\nvm\current\nodejs\nodejs\npm.CMD
    pnpm: 8.13.1 - ~\AppData\Local\pnpm\pnpm.CMD
    bun: 1.0.21 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (121.0.2277.83)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @auth/drizzle-adapter: ^0.4.0 => 0.4.0 
    next: ^14.0.4 => 14.1.0 
    next-auth: ^4.24.5 => 4.24.5 
    react: 18.2.0 => 18.2.0 

### Reproduction URL

https://github.com/xkalf/nomad_competition

### Describe the issue

```
⨯ Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\Users\hitech\code\nomad_competition\node_modules.pnpm@auth+drizzle-adapter@0.4.0\node_modules@auth\drizzle-adapter\lib\utils' imported from C:\Users
\hitech\code\nomad_competition\node_modules.pnpm@auth+drizzle-adapter@0.4.0\node_modules@auth\drizzle-adapter\lib\pg.js
at new NodeError (node:internal/errors:405:5)
at finalizeResolution (node:internal/modules/esm/resolve:327:11)
at moduleResolve (node:internal/modules/esm/resolve:946:10)
at defaultResolve (node:internal/modules/esm/resolve:1132:11)
at nextResolve (node:internal/modules/esm/loader:163:28)
at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
at ModuleWrap. (node:internal/modules/esm/module_job:77:40)
at link (node:internal/modules/esm/module_job:76:36) {
code: 'ERR_MODULE_NOT_FOUND'
}
```

### How to reproduce

pnpm dev and go to the localhost:3000 in server console is getting error calling db request is also not woking for this bug

### Expected behavior

i just want work well without the error

--- a/packages/adapter-drizzle/src/lib/pg.ts
+++ b/packages/adapter-drizzle/src/lib/pg.ts
@@ -10,7 +10,7 @@ import {
 } from "drizzle-orm/pg-core"
 
 import type { Adapter, AdapterAccount } from "@auth/core/adapters"
-import { stripUndefined } from "./utils"
+import { stripUndefined } from "./utils.js"
 
 export function createTables(pgTable: PgTableFn) {
   const users = pgTable("user", {

--- a/packages/adapter-drizzle/src/lib/sqlite.ts
+++ b/packages/adapter-drizzle/src/lib/sqlite.ts
@@ -7,7 +7,7 @@ import {
   BaseSQLiteDatabase,
   SQLiteTableFn,
 } from "drizzle-orm/sqlite-core"
-import { stripUndefined } from "./utils"
+import { stripUndefined } from "./utils.js"
 
 import type { Adapter, AdapterAccount } from "@auth/core/adapters"
 

